### PR TITLE
[VEN-2385]: liquidate exploiter debt and send BNBs to Binance

### PR DESCRIPTION
**Pull request by kkirka**
This VIP partially liquidates the exploiter debt using vUSDC from treasury, and sends the redeemed BNBs to Binance wallet for swapping into USDT/USDC. More repayments will be done in the consecutive VIPs.

In detail, this VIP:

* sets a temporary Liquidator contract, so that the entire 10% incentive is seized during the liquidation
* withdraws vUSDC from treasury to the TokenRedeemer contract
* invoke redeemAndTransfer in the TokenRedeemer contract, sending 100% of the withdrawn USDC to the temporary liquidator
* invokes liquidateAndRedeemNative function of the temporary contract, that would, in turn, liquidate the position, redeem vBNBs and transfer them to the Binance wallet
* resets the address of the Liquidator contract

**Note** that there are two known potential DoS vectors:

1. insufficient liquidity in BNB or USDC market may block redemptions, making the VIP fail
2. a huge USDC donation to the temporary contract may make it try to liquidate more than the exploiter's debt, which would make the VIP fail

(1) is out of the direct control of the community or development team, while (2) is unlikely, and would rather benefit the protocol if attempted.

### CODE REVIEW

File: simulations/vip-254/simulations.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+const TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+const LIQUIDATE_AND_REDEEM_HELPER = "0xA08301b7C5f4BccD654De95E8C9BD4388CC54Ec1";
+const VBNB = "0xA07c5b74C9B40447a954e1466938b865b6BBea36";
+const LIQUIDATOR = "0x0870793286aaDA55D39CE7f82fb2766e8004cF43";
+const COMPTROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+
+const EXPLOITER_WALLET = "0x489A8756C18C0b8B24EC2a2b9FF3D4d447F79BEc";
+const BINANCE_WALLET = "0x6657911F7411765979Da0794840D671Be55bA273";
+const REPAY_AMOUNT = parseUnits("39137.18", 18);
+const EXPECTED_BNB_AMOUNT = parseUnits("131.130431007685944719", 18);
```
Comment (GitGuru7) [inline simulations/vip-254/simulations.ts:26]: Constants can be reused by importing from the VIP implementation.

**Review by GitGuru7 — APPROVED** [review]
Simple comment but otherwise LGTM

File: simulations/vip-254/simulations.ts (hunk 1 above)
Comment (kkirka) [reply simulations/vip-254/simulations.ts]: Idk, I still prefer to not import implementation to tests, it kinda allows me to notice the exact tests that break when I change the impl

**Review by chechu — APPROVED** [review]

**Comment by kkirka** [comment]
The account is [healthy](https://debank.com/profile/0x489A8756C18C0b8B24EC2a2b9FF3D4d447F79BEc) atm, no need for liquidation unless BNB price drops significantly or interest accumulates for long enough. This PR may be reopened (with a different VIP number) later if the need arises.

**Pull request closed without merging**